omnipaxos/src/omni_paxos.rs

```diff
@@ -73,18 +69,77 @@ impl OmniPaxosConfig {
     }
 }
 
-impl Default for OmniPaxosConfig {
+/// Configuration for an `OmniPaxos` cluster.
+/// # Fields
+/// * `configuration_id`: The identifier for the cluster configuration that this Sequence Paxos replica is part of.
+/// * `nodes`: The nodes in the cluster i.e. the `pid`s of the other replicas in the configuration.
+/// * `initial_leader`: The initial leader of the cluster. Could be used in combination with reconfiguration to skip the prepare phase when switching to a new configuration.
```

Comment:
Since we call the other config `ServerConfig`, it is better if our docs here say "server" instead of "replica". And also "OmniPaxos" instead of "Sequence Paxos".

Reply:
I'm not a fan of the name `ServerConfig`, to me it implies the whole network layer as well. How about `ReplicaConfig`?